NEWS.md

```diff
@@ -20,13 +20,16 @@ For example:
 > - [bytecode_api.h](headers/bytecode_api.h)
 > - [bytecode_local.h](headers/bytecode_local.h)
 
-## `0.103.1`
+## `0.105.0`
 
 ➕ Complete overhaul of the bytecode compiler project.
   - The clamav compiler passes now build against a system-installed LLVM-
     library. This builds a single shared library (i.e. libclambcc.so).
   - The compiler application is now a Python script that uses Clang with the
     bytecode compiler library to run the compiler passes.
+  - Fixed bug causing signatures with malloc to fail to run when using system
+    installed llvm.
+  - Fixed the peinfo.c example bytecode source.
```

Comment:
These should have their own separate lines and use the 🐛-emoji to indicate that they are bug-fix changes, instead of being bullets on the overhaul "new feature" change.